easy_switch_user/views/res_users_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version='1.0' encoding='UTF-8' ?>
+<odoo>
+
+   <record id="view_users_form" model="ir.ui.view">
+        <field name="name">res.users.form.inherit</field>
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.view_users_form" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <button
+                    type="object"
+                    name="action_switch_user"
+                    aria-label="Switch User"
+                    string="Switch User"
+                    groups="base.group_system"
+                    attrs="{'invisible': &quot;[('id', '=', uid)]&quot;}"
```

Comment:
Not need to escape double quotes, just use single quotes instead of the escaped ones

Reply:
It did not work, actually, I saw a lot of button in this way as you can see [here](https://github.com/odoo/odoo/blob/25bc19dc1351b64505d8a2d41dd2b90afd7c117a/addons/auth_totp/views/res_users_views.xml#L30)